internal/pkg/config/config.go

```diff
@@ -268,33 +268,38 @@ func (pm packageMatcher) match(pkg *types.Package) bool {
 	return pm.PackageNameRE.MatchString(pkg.Path())
 }
 
-type NameMatcher struct {
-	PackageRE regexp.Regexp
-	TypeRE    regexp.Regexp
-	MethodRE  regexp.Regexp
+type methodMatcher struct {
+	PackageRE  regexp.Regexp
+	ReceiverRE regexp.Regexp
+	MethodRE   regexp.Regexp
 }
 
-func (r NameMatcher) matchPackage(p *types.Package) bool {
+func (r methodMatcher) matchPackage(p *types.Package) bool {
 	return r.PackageRE.MatchString(p.Path())
 }
 
-func (r NameMatcher) MatchMethodName(c *ssa.Call) bool {
-	if c.Call.StaticCallee() == nil || c.Call.StaticCallee().Pkg == nil {
+// Match matches methods based on package, method, and receiver regexp.
+// To explicitly match a method with no receiver (i.e., a top-level function),
+// provide the ReceiverRE regexp `^$`.
+func (r methodMatcher) Match(c *ssa.Call) bool {
+	callee := c.Call.StaticCallee()
+	if callee == nil || callee.Pkg == nil {
```

Comment:
It hadn't occurred to me yet to do this. Stealing this trick for future use. :smile: